packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.ts

```diff
@@ -148,6 +148,32 @@ const OverrideResolverLoader = function (this: LoaderContext<any>) {
     })
 }
 
+/**
+ * Extracts the absolute path to an extension package root from a source file path.
+ *
+ * @param source - The source file path to analyze
+ * @returns The absolute path to an extension package root if found, empty string otherwise
+ */
+const getExtensionPath = (source: string): string => {
+    const isMonoRepoPath = source.includes(`${path.sep}${MONO_REPO_WORKSPACE_FOLDER}${path.sep}`)
+    const isNodeModulesPath = source.includes(`${path.sep}${NODE_MODULES_FOLDER}${path.sep}`)
+
+    if (!isMonoRepoPath && !isNodeModulesPath) {
+        return ''
```

Comment:
I'd recommend that you return `undefined` here because an empty string could be a little ambiguous since some systems consider it a valid path and others don't. In short, it's probably not the clearest and most semantically correct choice for this return value.

Reply:
The guard `if (!packagePath)` will catch both empty strings and undefined and return false in either case. 
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/0560bf4ccb6fcf3c4eb0acbb74e6cd1c91e54a71/packages/pwa-kit-extension-sdk/src/shared/utils/extensibility.ts#L105
But you're right that returning an empty string '' can be consider a truthy value in some context, so I updated to return directly undefined.